src/dmd/dsymbolsem.d

```diff
@@ -2743,44 +2743,44 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
          */
         if (f.isscope && !funcdecl.isNested() && !ad)
         {
-            funcdecl.error("functions cannot be scope");
+            funcdecl.error("functions cannot be `scope`");
         }
 
         if (f.isreturn && !funcdecl.needThis() && !funcdecl.isNested())
         {
             /* Non-static nested functions have a hidden 'this' pointer to which
              * the 'return' applies
              */
-            funcdecl.error("static member has no `this` to which `return` can apply");
+            funcdecl.error("`static` member has no `this` to which `return` can apply");
         }
 
         if (funcdecl.isAbstract() && !funcdecl.isVirtual())
         {
             const(char)* sfunc;
             if (funcdecl.isStatic())
-                sfunc = "static";
+                sfunc = "`static`";
             else if (funcdecl.protection.kind == Prot.Kind.private_ || funcdecl.protection.kind == Prot.Kind.package_)
                 sfunc = protectionToChars(funcdecl.protection.kind);
             else
-                sfunc = "non-virtual";
-            funcdecl.error("%s functions cannot be abstract", sfunc);
+                sfunc = "`final`";
+            funcdecl.error("%s functions cannot be `abstract`", sfunc);
```

Comment:
Does protectionToChars return backticked string? I was thinking ```"`%s`" function..."```

Reply:
Fix up can be done in next pr though.

Reply:
Yeah, I tried to do that, but `protectionToChars` is also used in one of the tests in the test suite to generate code, so I couldn't make the change without clobbering that test.  I have to find another way.  The `.error` and `.deprecation` methods also automatically prepend the `.toChars()` to the error message, and none of those strings are highlighted yet.  This is all coming in a future PR, but I've got to do some figuring first.

Reply:
I moved the tick marks to the error message in #7763.